src/Tabs/Tabs.component.tsx

```diff
@@ -94,12 +94,12 @@ export const Tabs = ({
     onChange,
     ...props
 }: TabsProps): React.ReactElement<TabsProps> => {
-    let tabInfo: Array<{
```

Comment:
why move from `Array` of blah to this other notation?

Reply:
Hah, I actually kinda hate this. It's another lint rule that I enabled. While I hate this, the Typescript community is more widely swayed by the `[]int` syntax than `Array<int>` and the most compelling reason is that `Array` is a JS language construct that is being used by TS as an interface which is "confusing". I'm cool switching this back because I think it reads better, but this seems to be more widely adopted by the TS community, especially those in project we borrow capital from.